.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: release
+
+permissions:
+  contents: write
+  packages: write
+  statuses: write
+  pull-requests: read
+
+on:
+  # Allows you to run this workflow manually from the 'Actions' tab.
+  workflow_dispatch:
```

Comment:
What happens when you run this workflow manually? What happens if you run it twice for the same branch, commit, or tag?

Reply:
Yeah, this should only really be run once for a given tag. In the event that it's run twice and the release hasn't been removed, the operation will fail. I'll go ahead and remove this.